syntax = "proto3";

package fixbug;

option cc_generic_services = true;


message GetFriendsListRequest {
    uint32 id = 1;
}

message GetFriendsListResponse {
    bool success = 1;
    repeated bytes frieds = 2;

}

service GetFriendsListServiceRPC {
    rpc GetFriendsList(GetFriendsListRequest) returns(GetFriendsListResponse);
}